### ISSUE TITLE
Remove number input from Figma

### ISSUE THREAD

**Issue by micahchiang** [department-of-veterans-affairs/vets-design-system-documentation#2616]
## Description
In addition to #2615, references to number input need to be removed from Figma so designers know they no longer need to use it. 

## Tasks
- [ ] Remove any references to `va-number-input` from Figma

## Acceptance Criteria
- [ ] Above tasks have been completed

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2616?planning-poker) with Zenhub @danbrady @LWWright7

**Comment by danbrady** [comment]
After our discussion, we decided not to remove the component completely from Figma just yet. We added a "Deprecated" note with suggested alternate component to the component in case a designer updates existing design or attempts to use it in a new design.

@LWWright7 @babsdenney This is ready for [review](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=1964-39107&t=DE2LmzEr7lVyUIKw-1).
